### DISCUSSION TITLE
Bill Splitting (Percentage or dollar based)

### DISCUSSION THREAD

**Discussion by buckchoklit** [firefly-iii/firefly-iii#8420] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

- Add the ability to bill split

### Solution

- People in married households that pay bills together but otherwise keep track of finances separately would be able to split their bills (percentage based, dollar based, etc).

### Additional context

My wife and I split bills by percentage of income. The way Firefly is today, one cannot correctly keep track of a liability where the expense is shared. For example, let's say I create a liability for my mortgage of $300,000. My monthly mortgage is $2400, but based on how my wife and I split our expenses, I pay $1,320. If I were to set up a bill for $1,320 and create a transaction with the destination being the liability, only $1,320 would be deducted from it instead of the $2400.

I would like to be able to input the total bill as $2400, but have a way to express my responsibility of the bill as a percentage or dollar amount (in this case $1,320) so that when the expense is tracked the $1,320 is withdrawn from the correct asset account, but the $2400 is deducted from the liability.

**Comment by JC5** [comment]
You can already do that, you don't need a new feature for that. But the money has to come from somewhere, it's up to you where.

1. Create a new shared asset account where your partner's money is deposited in as well (for real or not), and use it as a source to pay the mortgage.
2. Create your own payment of 1320, and create a second payment from a (fake) expense account or asset account for the rest of the money.
